src/constants.ts

```diff
@@ -7,3 +9,24 @@ export const OPENSEA_BASE_NFTS_URL = `${OPENSEA_BASE_URL}/assets`;
 export const MAGICEDEN_BASE_URL = 'https://magiceden.io';
 export const MAGICEDEN_BASE_COLLECTIONS_URL = `${MAGICEDEN_BASE_URL}/marketplace`;
 export const MAGICEDEN_BASE_NFTS_URL = `${MAGICEDEN_BASE_URL}/item-details`;
+
+const evmLabels = {
+    collectionId: "Contract Address",
+    tokenId: "Token ID",
+    tokenStandard: "ERC721"
+};
+
+/**
+ * Labels for collection, token, and token standards per-chain.
+ * eg: on EVM chains, we have a "Contract Address", but on Solana that's
+ * referred to as an "On-chain Collection"
+ */
+export const chainIdentifierLabels: Record<Chain, { collectionId: string; tokenId: string; tokenStandard: string; }> = {
```

Comment:
should `chainIdentifierLabels` be uppercase snake case?

Reply:
my personal approach is always (i think) uppercase snake case for primitive consts, camelCase for arrays / objects / mappings etc. Not super opinionated tho

Reply:
it also calls into question some of the stuff in `utils.ts`, as its kind of also a constant that just happens to have functions defined within it